views/mesh.py

```diff
@@ -3,61 +3,34 @@
 from flask_admin import form
 from wtforms import ValidationError
 
-# class MeshUploadField(form.FileUploadField):
-#     def pre_validate(self, form):
-#         if self._is_uploaded_file(self.data) and not self.is_file_allowed(
-#             self.data.filename
-#         ):
-#             raise ValidationError("Invalid file extension")
-#
-#     def _delete_file(self, filename):
-#         return filename
-#
-#     def populate_obj(self, obj, name):
-#         if self._is_uploaded_file(self.data):
-#             filename = self.generate_name(obj, self.data)
-#             filename = self._save_file(self.data, filename)
-#             # update filename of FileStorage to our validated name
-#             self.data.filename = filename
-#             setattr(obj, name, filename)
-#             setattr(obj, "file_bucket", self.base_path)
-#
-#     def _save_file(self, data, filename, base_path="."):
-#
-#         s3.Bucket(bucket).Object(self.data.filename).put(Body=data.read())
-#
-#         return self.data.filename
-
-
 class MeshView(UserModelView):
     can_edit = False
     column_list = (
         Mesh.id,
+        "project",
+        Mesh.latitude,
+        Mesh.longitude,
         Mesh.name,
-        Mesh.file_name,
-        Mesh.centroid_x,
-        Mesh.centroid_y,
+        Mesh.zipfile,
+        "odmproject",
         Mesh.status,
     )
 
     column_labels = {
         "name": "Mesh name",
-        "centroid_x": "Longitude [deg.]",
-        "centroid_y": "Latitude [deg.]",
-        "file_name": "Zip file"
+        "zipfile": "Zip file"
     }
     column_descriptions = {
-        "name": "Your name for the project",
-        "n_cams": "Amount of cameras available",
-        "dt": "Zip file containing mesh data",
     }
     form_columns = (
+        "project",
+        Mesh.latitude,
```

Comment:
This is a bit risky. At some point we will want to have a lot of detail and precision on the locations of elements within the mesh, whereas this is just to place it on a big world map for selecting a mesh area. 

Might make sense to explicitly name this something like ```Mesh.centroid_latitude``` to make sure it's obvious that it's just an approximate location?